docs/judge/connecting_a_judge.md

```diff
@@ -0,0 +1,36 @@
+# Connecting a Judge to the Site
+
+This guide is for connecting a judge to the site.
+
+## Site-Side
+
+First, add a judge on the admin page, located at [https://dmoj.ca/admin/judge](https://dmoj.ca/admin/judge). Provide the name of the judge and the authentication key for the judge. You may use the `Regenerate` button to generate an authentication key.
+
+In `local_settings.py`, find the `BRIDGED_JUDGE_ADDRESS`. This is the address you will be connecting a judge to. By default, this should be `localhost:9999`. If you are connecting from a different machine, you will need to change `localhost` to an actual IP. Also, ensure that this port is **open**, or you will receive cryptic error messages when attempting to connect a judge.
+
+Finally, ensure the bridge is running. You should see something similar to the following lines.
+```shell-session
+$ supervisorctl status
+bridged RUNNING pid <pid>, uptime <uptime>
+```
+
+## Judge-Side
+
```

Comment:
We should shepherd people into using the Docker variant of this, or at least present both (with Docker first). (See judge's README.md).